integration-commons/src/main/java/org/symphonyoss/integration/model/yaml/TimeoutConfig.java

```diff
@@ -0,0 +1,45 @@
+package org.symphonyoss.integration.model.yaml;
+
+/**
+ * Holds the timeout configurations.
+ * Created by campidelli on 9/22/17.
+ */
+public class TimeoutConfig {
```

Comment:
We should try to reuse the 'ApiClientConfig' class. In fact, we must rename this class to another one like HttpClientConfig or something like that.